release.py

```diff
@@ -172,7 +172,7 @@ def make_release_branch(release_set, release_version, release_project):
     execute_commands(
         [
             'git checkout -b {branch} --no-track origin/develop',
-            'echo {release_version} | {release_set}',
+            '{release_set} {release_version}',
```

Comment:
The thing was actually working. This piped thing is needed to support different variations (like in a native app):
```
# web
xargs -I {} npm run update-version -- --next-version {} && npm i;
# native
xargs -I {} yarn version --no-git-tag-version --new-version {} && npm i;
```